tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/TaskProblemAction.java

```diff
@@ -59,9 +60,9 @@ public void run( final TaskProblem taskProblem, final ISelectionProvider provide
     {
         final IFile file = MigrationUtil.getIFileFromTaskProblem( taskProblem );
 
-        new Job( "Marking migration problem as done" )
+        new UIJob( "Marking migration problem as done" )
```

Comment:
Because we will remove the table data for problemView in IgnoreAction class, it will be run int UI thread. I have test it with normal Job, it will cause invliad thread exception